src/provider_references.jl

```diff
@@ -0,0 +1,74 @@
+import Tokenize.Tokens
```

Comment:
sticking to public api's as much as you can should theoretically be more stable, you'd hope